packages/ses/test/error/test-tame-v8-error-unsafe.js

```diff
@@ -51,6 +51,28 @@ function simulateDepd() {
   return site.name;
 }
 
+test('SES compartment error compatibility', t => {
```

Comment:
Since the problem this test illustrates has not been fixed yet, if merged as is it will cause failures. However, if you change it to `test.fail`, then IIUC, it will pass until the problem has been fixed. Please also file an issue explaining the problem and link it from the test case with a comment.

```suggestion
// link to issue
test.fail('SES compartment error compatibility', t => {
```